nucliadb/tests/nucliadb/integration/test_conversation.py

```diff
@@ -320,3 +328,222 @@ async def test_cannot_create_message_with_slash(
+    question = "What is the meaning of life?"
+    vectors[question] = [1.0] * 768
+
+    slug = "myresource"
+    resp = await nucliadb_writer.post(
+        f"/kb/{kbid}/resources",
+        json={
+            "slug": slug,
+            "title": "My Resource",
+            "conversations": {
+                "faq": {
+                    "messages": [
+                        {
+                            "to": ["computer"],
+                            "who": "person",
+                            "timestamp": datetime.now().isoformat(),
+                            "content": {"text": question},
+                            "ident": "1",
+                            "type": MessageType.QUESTION.value,
+                        },
+                    ]
+                },
+            },
+        },
+    )
+    resp.raise_for_status()
+    rid = resp.json()["uuid"]
+
+    # Inject synthetic processed data for the conversation
+    await inject_message(
+        nucliadb_ingest_grpc, _broker_message(split="1", text=question, vector=vectors[question])
+    )
+    await mark_dirty()
+    await wait_for_sync()
+
+    # Check counters
+    counters = await get_counters()
+    assert counters.sentences == 1  # One for the question message
+    assert counters.paragraphs == 2  # One for the question message + title paragraph
+    assert counters.fields == 2  # conversation field + title field
+    assert counters.resources == 1
+
+    # Append a message
+    answer = "42"
+    vectors[answer] = [2.0] * 768
+    resp = await nucliadb_writer.put(
+        f"/kb/{kbid}/resource/{rid}/conversation/faq/messages",
+        json=[
+            {
+                "to": ["person"],
+                "who": "computer",
+                "timestamp": datetime.now().isoformat(),
+                "content": {"text": answer},
+                "ident": "2",
+                "type": MessageType.ANSWER.value,
+            }
+        ],
+    )
+
+    await inject_message(
+        nucliadb_ingest_grpc, _broker_message(split="2", text=answer, vector=vectors[answer])
+    )
+
+    await mark_dirty()
+    await wait_for_sync()
+
+    # Check counters after appending the message
+    counters = await get_counters()
+    assert (
+        counters.sentences == 2 + 1
+    )  # One for each message (the title does not have a vector) + the initial message that has been marked as deleted but not yet merged
+    assert counters.paragraphs == 3  # One for each message + the title paragraph
+    assert counters.fields == 2  # One conversation field + the title field
+    assert counters.resources == 1
+
+    # Make sure the messages are searchable
+    question_text_block_id = f"{rid}/c/faq/1/0-{len(question)}"
+    results = await search_message(query=question)
+    assert len(results.best_matches) == 1
+    assert question_text_block_id in results.best_matches
+
+    # TODO: Look into why both messages of the conversation get the same vector score!
```

Comment:
@javitonino tagging you here so you get the github email and we look into it together once you are back